justfile

```diff
@@ -9,8 +9,8 @@ install:
   pip install --no-deps -r requirements-dev.txt
 
 fmt:
-  ruff check --fix ./ruff_lsp ./tests
   ruff format ./ruff_lsp ./tests
+  ruff check --fix ./ruff_lsp ./tests
```

Comment:
What's the reason for changing the order here? Fixing should run before formatting to avoid that fixes introduce new formatting errors. Or did you run into situations where the formatter produced new lint violations, if so, that's a bug (except if it is a rule that this known to be incompatible with the formatter)

Reply:
I was getting annoyed because the formatter wouldn't run if any checks failed (like E501). But the right fix is to use `--fix-only`, changing...

(This setup is a vestige from before the formatter existed.)

Reply:
https://github.com/astral-sh/ruff-lsp/pull/369